plugins/dragonframe/plugin.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../../schemas/plugin.json",
+
+  "name": "Dragonframe",
+  "description": [
+    "<p><em>Dragonframe</em> is software for stop motion filmmaking, with DMX programming capabilities.",
+    "Dragonframe 5 will support lighting fixtures.</p>"
+  ],
```

Comment:
Please also mention the OFL format version that this plugin is based on here. See the [Millumin](https://open-fixture-library.org/about/plugins/millumin) description.